android/src/main/java/com/rnziparchive/RNZipArchivePackage.java

```diff
@@ -16,6 +16,12 @@
     modules.add(new RNZipArchiveModule(reactContext));
     return modules;
   }
+  
+  // deprecated >= RN 0.47.0
```

Comment:
Oh, I know this change early and still thinking how to make this compatible with earlier versions of rn.

Have you tested it on both 0.47 and <0.47? If it works I would merge it and ship it with next release.